flixel/FlxObject.hx

```diff
@@ -540,6 +549,17 @@ class FlxObject extends FlxBasic
 	 * Overriding this will force a specific color to be used for debug rect.
 	 */
 	public var debugBoundingBoxColor:Null<Int> = null;
+	
+	public var debugColorScheme(default, null):DebugColorScheme;
+	
+	/**
+	 * Can be overriden for RenderBlit mode to re-render stuff with new colors
+	 */
+	public function setDebugColorScheme(debugColorScheme:DebugColorScheme)
```

Comment:
@starry-abyss so do you agree to add 3 debug color properties instead of `sebugColorScheme`?